db/migrate/20211118145604_alter_structure_of_neighbourhoods.rb

```diff
@@ -0,0 +1,99 @@
+class AlterStructureOfNeighbourhoods < ActiveRecord::Migration[6.0]
```

Comment:
The main migration, the gist of the update is "shove things into the new values". We create new districts, counties, and regions based on the data available in the previous neighbourhoods, but in some cases (i.e. `region`), the information was present as the `ward.region` attribute, but not in the code data. So we just use those attributes, as they seemed to be more accurate than the code attributes anyway.